elastic-utils-lib/src/cmr/elastic_utils/embedded_elastic_server.clj

```diff
@@ -1,15 +1,19 @@
 (ns cmr.elastic-utils.embedded-elastic-server
-  "Used to run an Elasticsearch server inside an embedded docker container."
+  "Used to run Elasticsearch servers inside an embedded docker container."
   (:require
    [cmr.common.lifecycle :as lifecycle]
    [cmr.common.log :as log :refer [debug error]]
    [cmr.common.util :as util])
   (:import
    (java.time Duration)
+   (java.util.function Consumer)
    (org.testcontainers.containers FixedHostPortGenericContainer Network)
    (org.testcontainers.containers.wait.strategy Wait)
    (org.testcontainers.images.builder ImageFromDockerfile)))
 
+(def DEFAULT_MEMORY_BYTES
+  (* 4 1024 1024 1024))
+
 (def ^:private elasticsearch-official-docker-image
   "Official docker image."
   "docker.elastic.co/elasticsearch/elasticsearch:7.17.14")
```

Comment:
has anyone ever tried 7.17.28? https://hub.docker.com/layers/library/elasticsearch/7.17.28/ it has less vulnerabilities